randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -427,6 +427,9 @@ Extra - room_id: [13]
   * Extra - door_idx: (29,)
   > Room Center
       Trivial
+  > Event - Stabilizer 3 Online
+      # Speed Boost the Stabilizer: https://youtu.be/V20W0F4dZP0
+      Level 0 Locks and Speed Booster and After Stabilizer 4 Restored and Shinespark Tricks (Intermediate) and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
you can do this without doing stablizer 4. im not sure if you have to kill the moto during running, or if you can kill it before hand.

Reply:
nvm ignore, mustve mistook it where its possible with going l->r

Reply:
I'm not sure you can. There's not enough room when running right to left to get the shinespark. Oddly, slopes do matter for this and you need more room running down a slope than running up one I think